multicluster/controllers/multicluster/commonarea/resourceimport_controller.go

```diff
@@ -364,11 +367,12 @@ func getMCService(resImp *multiclusterv1alpha1.ResourceImport) *corev1.Service {
 	return mcs
 }
 
-func getMCServiceImport(resImp *multiclusterv1alpha1.ResourceImport) *k8smcsv1alpha1.ServiceImport {
+func getMCServiceImport(resImp *multiclusterv1alpha1.ResourceImport, clusterID string) *k8smcsv1alpha1.ServiceImport {
 	svcImp := &k8smcsv1alpha1.ServiceImport{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      resImp.Spec.Name,
-			Namespace: resImp.Spec.Namespace,
+			Name:        resImp.Spec.Name,
+			Namespace:   resImp.Spec.Namespace,
+			Annotations: map[string]string{"localClusterID": clusterID},
```

Comment:
Can we use format similar to Antrea and K8s's well know annotations? There should be a prefix defining its scope, and words should be connected with "-". For instances:
node.antrea.io/mac-address
service.antrea.io/external-ip-pool
service.kubernetes.io/topology-aware-hints

Reply:
done, add a new annotation const.

Reply:
forget to update?

Reply:
done, fix it and unit test. thanks.